src/Confluent.Kafka/Consumer.cs

```diff
@@ -620,6 +620,27 @@ public Metadata GetMetadata(bool allTopics, TimeSpan timeout)
         /// </summary>
         public Metadata GetMetadata(bool allTopics)
             => consumer.GetMetadata(allTopics);
+
+        /// <summary>
+        ///     Adds one or more brokers to the Consumer's list of initial
+        ///     bootstrap brokers. 
+        ///
+        ///     Note: Additional brokers are discovered automatically as 
+        ///     soon as the Consumer connects to a broker by querying the 
+        ///     broker metadata. Calling this method is only required in 
+        ///     some scenarios where the address of all brokers in the 
+        ///     cluster changes.
```

Comment:
Maybe add a note that there currently is no API to remove existing configured, added or learnt brokers.